src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet8

```diff
@@ -6,8 +6,8 @@ RUN cd /src/dotnet-function-app && \
     dotnet publish *.csproj --output /home/site/wwwroot
 
 # To enable ssh & remote debugging on app service change the base image to the one below
-# FROM mcr.microsoft.com/azure-functions/dotnet:4-dotnet8-appservice
-FROM mcr.microsoft.com/azure-functions/dotnet:4-dotnet8
+# FROM mcr.microsoft.com/azure-functions/dotnet:4-dotnet8.0-appservice
```

Comment:
please add a script that does docker pull and verifies image listed in the dockerfile is valid - sperate PR OK - open new issue and add link here